packages/components/src/input-control/styles/input-control-styles.tsx

```diff
@@ -213,7 +213,7 @@ export const Input = styled.input< InputProps >`
 		box-sizing: border-box;
 		border: none;
 		box-shadow: none !important;
-		color: ${ COLORS.gray[ 900 ] };
+		color: ${ COLORS.theme.foreground };
```

Comment:
Unrelated to borderless, but I'd like to sneak in this small enhancement for better themability if that's ok 🥺